src/components/Home.js

```diff
@@ -0,0 +1,38 @@
+import classes from '../Root.module.css';
+
+const Homepage = () => {
+  return (
+    <>
+      <h1>Welcome to our page!</h1>
+      <p className={classes.paragraph}>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut ut ligula
+        aliquam, eleifend felis ut, dapibus ipsum. Sed eleifend nunc vel posuere
+        rhoncus. Mauris vitae nisi quam. Vivamus non risus dapibus, commodo ex
+        id, posuere justo. Donec nec enim purus. Aenean sit amet leo dapibus,
+        placerat libero et, lacinia turpis. Pellentesque habitant morbi
+        tristique senectus et netus et malesuada fames ac turpis egestas. In
+        tincidunt, odio et pulvinar congue, elit sem finibus massa, vel laoreet
+        elit erat vitae orci. Sed sit amet ultricies eros. Nam bibendum quam
+        vitae turpis interdum, id bibendum turpis fermentum. Etiam rutrum
+        dapibus lacinia. Integer sed lorem lectus. Fusce commodo felis orci, a
+        consequat orci fermentum et. Ut quis nisl lacinia, iaculis ante ut,
+        pretium lacus. Ipsum.
+      </p>
+
+      <p className={classes.paragraph}>
+        Lorem ipsum Nulla porttitor ullamcorper elit, in iaculis sem aliquet a.
+        Curabitur tincidunt augue vitae tortor elementum lobortis. Proin
+        fringilla ex vitae sapien sagittis, eu porttitor nisi finibus. Nunc ut
+        metus eu mi scelerisque fermentum. Duis sagittis ultricies semper. Sed
+        pulvinar, tortor vel volutpat aliquam, nisi lorem eleifend neque, in
+        malesuada dolor nisi ut magna. Nullam cursus mi in tellus malesuada
+        rhoncus. Proin a diam eleifend, dictum massa at, congue est. Phasellus
+        auctor lectus ipsum, eu efficitur metus eleifend non. Fusce feugiat
+        semper dolor ac malesuada. Nunc sodales, orci in feugiat tincidunt,
+        tellus tellus suscipit velit, eget tincidunt eros turpis non tortor.
+        Ipsum.
+      </p>
```

Comment:
Good job by creating this view 👍🏼 , but please use your own text to make your work look professional it is not good to deploy your app with auto generated text